### PR TITLE
Add recipes for opentelemetry

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
Recipe generated using grayskull

File: recipes/opentelemetry-api/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "opentelemetry-api" %}
+{% set version = "0.12b0" %}
```
Comment (scopatz) [inline recipes/opentelemetry-api/meta.yaml:2]: Are there full releases @mariusvniekerk? Otherwise this should follow CFEP5

Comment (mariusvniekerk) [reply recipes/opentelemetry-api/meta.yaml:2]: there aren't yet.  They seem to be using the b tag to prevent automatic installation until they hit a 1.0

Comment (mariusvniekerk) [reply recipes/opentelemetry-api/meta.yaml:2]: https://pypi.org/project/opentelemetry-api/#history

Comment (scopatz) [reply recipes/opentelemetry-api/meta.yaml:2]: So I think this is blocked then

Comment (mariusvniekerk) [reply recipes/opentelemetry-api/meta.yaml:2]: It's basically the same as `black` which has also never had a non-beta release

**Pull request merged**